ghost/core/core/server/api/endpoints/links.js

```diff
@@ -0,0 +1,26 @@
+const linksService = require('../../services/link-redirection');
+const linkTrackingService = require('../../services/link-click-tracking');
+
+module.exports = {
+    docName: 'links',
+    browse: {
+        options: [
+            'filter'
+        ],
+        permissions: false,
+        async query(frame) {
+            const links = await linkTrackingService.service.getLinks(frame.options);
+
+            return {
+                data: links,
+                meta: {
```

Comment:
Not sure about hardcoding this here, should maybe come from the `getLinks` method?

Reply:
Lets keep it in the endpoint implementation for now. The `getLinks` method shouldn't really know the endpoint structure I think